processing/src/main/java/org/apache/carbondata/processing/surrogatekeysgenerator/csvbased/CarbonCSVBasedSeqGenMeta.java

```diff
@@ -111,7 +110,7 @@
   /**
    * timeFormat
    */
-  protected SimpleDateFormat timeFormat;
+  protected String dateFormat;
```

Comment:
better to complete modification for all method and keep the order.
e.g.  saveRep, readRep...